test/components/polylith/bricks/test_base.py

```diff
@@ -0,0 +1,37 @@
+from test import params
+
+import pytest
+
+from components.polylith.bricks.base import create_base, get_bases_data
+
+
+@pytest.mark.parametrize(
+    "id, expected_dirs, expected_dir_structure",
+    params.create_base_params,
+    ids=params.create_base_ids,
+)
+def test_create_base(handle_workspace_files, id, expected_dirs, expected_dir_structure):
```

Comment:
I understand the idea of the fixtures and params, but think it would be easier to follow if it was explicit in the unit test. I find myself jump back and forth to check what the data is about. Maybe the expected data could be in the unit test as variables?

Reply:
With only a couple tests, it's not a problem to do it like that, but I find that putting the params in with the test function makes the test function itself less readable, especially as more tests are added.